lib/config.js

```diff
@@ -144,6 +144,7 @@ Config.prototype.buildArgs = function () {
     is_component_build: this.isComponentBuild(),
     proprietary_codecs: true,
     ffmpeg_branding: "Chrome",
+    branding_path_component: "brave",
```

Comment:
In https://github.com/brave/brave-core/pull/6091/files?file-filters%5B%5D=.gni#diff-c88135de3a4f2ec8d5bd8011fef74b44L8 the branding wasn't getting set for iOS. We may want to delete this argument from iOS config further down in line 289.

Reply:
Oops! Very good point, thanks for catching that up. Fixed!